### ISSUE TITLE
TalkBack not announcing links in hybrid apps, works correctly in wab pages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [2php/eyes-free#478]
```
What steps will reproduce the problem?
1. Create an HTML 5 page with 3 links
2. Use phonegap to create an android app
3. Activate TalkBack 4.1.08 abd swipe, or Explore by Touch, to links

What is the expected output? What do you see instead?
1. Expectation is that Talkback will announce the link text and that it is a 
link.
2. Result is that TalkBack only reads the link text and does not announce it as 
link

What version of the product are you using? On what operating system?
Lollipop 5.1.1, TalkBack 4.1.08

Please provide any additional information below.

If I define the role "link" or "button" then it works as expected.


```

Original issue reported on code.google.com by `mcgheegr...@gmail.com` on 28 Aug 2015 at 2:59
